### PR TITLE
Docs: Updated section "6.3. Editing a Kafka user" -  QE comment #770

### PR DESCRIPTION
**Pull request by dmuntima**
### Type of change

- Documentation

### Description

Fix QE comment #770

### CODE REVIEW

File: documentation/book/proc-changing-kafka-user.adoc [hunk 1]
```diff
@@ -15,9 +15,15 @@ This procedure describes how to change the configuration of an existing Kafka us
 
 .Procedure
 
+You can edit the user either by preparing a YAML file containing the desired `KafkaUser` or by using {ProductPlatformName} tools.
```
Comment (ncbaratta) [inline documentation/book/proc-changing-kafka-user.adoc]: If this is an introduction it should be above the procedure - procedures should only be numbered steps.

File: documentation/book/proc-changing-kafka-user.adoc [hunk 2]
```diff
@@ -49,37 +55,46 @@ spec:
...
 [source,shell,subs=+quotes]
-oc edit kafkauser _<your-user-name>_
+oc apply -f _<your-file>_
++
+. Use the updated credentials from the `my-user` secret in your application.
 
-. Update the `KafkaUser` resource in {ProductPlatformName}.
+[start=1]
+
+.Editing the `KafkaUser` resource using {ProductPlatformName} tools
```
Comment (ncbaratta) [inline documentation/book/proc-changing-kafka-user.adoc]: Why do we have this added label in the procedure?  We might want to ask the mod docs team how to handle a procedure with 2 options.

Comment (dmuntima) [reply documentation/book/proc-changing-kafka-user.adoc]: @ncbaratta, we have two ways of editing a user. Hence, I have used the label to differentiate the same. I will send an email to the mod-docs list to check how to handle a procedure with 2 options.

Comment (scholzj) [reply documentation/book/proc-changing-kafka-user.adoc]: If it helps, we can just keep the `kubectl apply` / `oc apply` procedure. The `kubectl edit` / `oc edit` is applicable everywhere else as well but I think we list it only here.

This is a bit related to my rants for some other PRs ... the users know how to use this and the will anyway ignore the procedure and do `oc create` / `oc replace` / `oc apply` / `oc edit` based on what they are used to.

**Comment by dmuntima** [comment]
@ncbaratta, @scholzj, I have retained the kubectl apply / oc apply procedure and deleted the kubectl edit /oc edit procedure.

**Pull request merged**